app/policies/encryptable_policy.rb

```diff
@@ -31,4 +31,22 @@ def destroy?
   def team
     @record.team
   end
+
+  private
+
+  def transferring_encryptable?
+    @record.receiver_id.present? &&
+      receiver_personal_team_owner? &&
+      user_human? &&
+      @record.is_a?(Encryptable::File)
+  end
+
+  def user_human?
+    User.find(@record.receiver_id).is_a?(User::Human)
```

Comment:
combine with `@record.folder.team.owner.id` ?
so we could get rid of attr accessor `receiver_id`